packages/routes/routes.0.5.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Anurag Soni <anurag@sonianurag.com>"
+authors: [ "Anurag Soni <anurag@sonianurag.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anuragsoni/routes"
+bug-reports: "https://github.com/anuragsoni/routes/issues"
+dev-repo: "git+https://github.com/anuragsoni/routes.git"
+doc: "https://anuragsoni.github.io/routes/"
+tags: [ "router" "http" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.8"}
```
The `dune-package` file in your source code contains `(lang dune 1.8)`. This means this package requires at least dune 1.8.

Reply:
I will take a look since i'm not sure i actually need any features introduced by dune 1.8
I should be able to use a lower version as a constraint. I'll make a new PR.

Thank you for catching this!

Reply:
I've updated the PR with a dune version constraint. Thanks again!